introduction.md

```diff
@@ -1 +1,5 @@
-Hi!  My name is Zack and I am super excited to be here.  My favorite CodeSchool track is Javascript one!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?